packages/starlight/translations/he.json

```diff
@@ -0,0 +1,22 @@
+  {
+    "skipLink.label": "דלגו לתוכן",
+    "search.label": "חיפוש",
+    "search.shortcutLabel": "(לחצו / לחיפוש)",
+    "search.cancelLabel": "ביטול",
+    "search.devWarning": "לא ניתן להציג תצוגה מקדימה של האתר באופן מקומי.",
```

Comment:
The English version here says a bit more than this sentence:

```
Search is only available in production builds. \nTry building and previewing the site to test it out locally.
```

Could you update to be a bit closer so it specifies the option of building and previewing as well?